src/SuperDumpService/Controllers/HomeController.cs

```diff
@@ -215,7 +215,7 @@ public class HomeController : Controller {
 				|| file.Extension == ".json") {
 				return Content(System.IO.File.ReadAllText(file.FullName));
 			}
-			return File(System.IO.File.ReadAllBytes(file.FullName), "application/octet-stream", file.Name);
```

Comment:
Doesn't the stream that ReadOpen creates need to be Disposed?

Reply:
Ok, I am wrong. File() helper method will take care of disposal itself, when it's finished streaming out the result.